tox.ini

```diff
@@ -30,6 +30,8 @@ deps =
     lint,tests,functests: factory-boy
     lint,tests,functests: h-matchers
     lint,template: cookiecutter
+    pluggy
+    lint,tests: pytest-mock
```

Comment:
In our previous library templates `tox.ini` is a fully managed file (without any configurable sections). It looks like this is now a part injected in via `.cookiecutter/includes/tox/deps`.

For example if you do something like this:

```shell
$> diff h-api/tox.ini h-matchers/tox.ini 
61c61
<     {release,initialrelease}: git push git@github.com:hypothesis/h-api.git --follow-tags
---
>     {release,initialrelease}: git push git@github.com:hypothesis/h-matchers.git --follow-tags

```

The only difference is a templated in string for the `initialrelease` (which I think has been removed). This is achieved with declaring our test dependencies in `setup.cfg` and:

```
{tests,lint}: -e .[tests]
```

This feels a bit like inventing our own solution (tox templating) where a standard exists (`setup.cfg` `install_requires/tests_require`). What are the reasons for using one over the other?

Reply:
As [discussed in Slack](https://hypothes-is.slack.com/archives/C1MA4E9B9/p1657735739581789?thread_ts=1657627437.243309&cid=C1MA4E9B9) let's postpone this discussion until the review of the cookiecutter itself